nucypher/blockchain/eth/sol/source/contracts/UserEscrow.sol

```diff
@@ -108,17 +108,17 @@ contract UserEscrow is Ownable {
     function withdrawETH() public onlyOwner {
         uint256 balance = address(this).balance;
         require(balance != 0);
-        owner.transfer(balance);
-        emit ETHWithdrawn(owner, balance);
+        msg.sender.transfer(balance);
+        emit ETHWithdrawn(msg.sender, balance);
     }
 
     /**
     * @dev Fallback function send all requests to the target proxy contract
     **/
-    function () public payable onlyOwner {
+    function () external payable onlyOwner {
```

Comment:
Why the change from public to external here?

Reply:
because of breaking changes in v0.5.0
https://solidity.readthedocs.io/en/v0.5.0/050-breaking-changes.html

Reply:
I still don't get it though - doesn't this change the visibility of this function?

Reply:
yes, `external` is more strict than `public`. `public` means that method also can be called from this contract, but `fallback` function can be called only outside of the contract. So for `fallback` function `public` works like `external` and in 0.5+ only `external` is allowed